src/main/java/duke/DukeListener.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.command.Command;
+import duke.command.CommandExit;
+
+
+/**
+ * Waits for and processes a User Input
```

Comment:
"User Input" could be "user input". No reason to capitalize here.

Reply:
You are right!